iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -92,6 +99,11 @@
                   Types.TimestampType.withoutZone(), Types.StringType.get(), Types.BinaryType.get(),
                   Types.DecimalType.of(3, 1), Types.UUIDType.get(), Types.FixedType.ofLength(5),
                   Types.TimeType.get());
+  private static final Map<String, String> STATS_MAPPING = ImmutableMap.of(
+      StatsSetupConst.NUM_FILES, SnapshotSummary.TOTAL_DATA_FILES_PROP,
+      StatsSetupConst.ROW_COUNT, SnapshotSummary.TOTAL_RECORDS_PROP
+      // TODO: add ROW_COUNT -> TOTAL_SIZE mapping after iceberg 0.12 is released
```

Comment:
small typo: TOTAL_SIZE -> TOTAL_FILE_SIZE_PROP mapping

Reply:
Fixed it.